posthog/src/test/java/com/posthog/internal/PostHogRemoteConfigTest.kt

```diff
@@ -69,7 +69,7 @@ internal class PostHogRemoteConfigTest {
     }
 
     @Test
-    fun `load flags from decide api and call the onFeatureFlags callback`() {
+    fun `load flags from decide api and call the onFeatureFlags callback - v3`() {
```

Comment:
Maybe we should move all 'flags' tests to a different file eg `PostHogFeatureFlagsV3Test.kt` and then v4? its more organized than a v3/v4 suffix in all tests.
This file would keep only the remote config tests which is a different API.

Reply:
Wil do!

Reply:
I'm thinking the current version will just be named `PostHogFeatureFlagsTest.kt`. If we ever release a v5, then I'll rename the test to `PostHogFeatureFlagsV4Test.kt` and `PostHogFeatureFlagsTest.kt` will become v5. That way, we can see the diff of how the tests change from V4 to V5.